sdk/textanalytics/ai-text-analytics/test/public/apiKey.spec.ts

```diff
@@ -124,6 +124,32 @@ describe("[API Key] TextAnalyticsClient", function() {
           const doc2Entity1Edge1Label = doc2Entity1.relatedEntities.values().next().value;
           assert.equal(doc2Entity1Target1.text, "ibuprofen");
           assert.equal(doc2Entity1Edge1Label, "DosageOfMedication");
+          assert.deepEqual(doc2.entityRelations[0], {
+            relationType: "DosageOfMedication",
+            roles: [
+              {
+                entity: doc2.entities[0],
```

Comment:
this looks great. just FYI that when the new model-version gets deployed u will have to update this test to the new roles